test/e2e/tools.mjs

```diff
@@ -162,7 +162,11 @@ async function copyPackages(dirName) {
     const destPackagesDir = join(dirName, 'packages');
     await fs.remove(destPackagesDir);
 
-    const { dependencies } = await fs.readJSON(join(dirName, 'package.json'));
+    const { dependencies, overrides } = await fs.readJSON(join(dirName, 'package.json'));
+
+    if (overrides?.apify) {
+        Object.assign(dependencies, overrides.apify);
+    }
```

Comment:
Ugly hack. This should be properly addressed when https://github.com/apify/crawlee/issues/1561 is fixed.